TLM/BUILDING_INSTRUCTIONS.md

```diff
@@ -0,0 +1,46 @@
+
+
+# Project building instructions
+
+
+### Prerequisites:
+* source code of this repository
+* source code of dependent repositories:
```

Comment:
In order to fetch the source and the submodules, ```git clone https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition``` and then cd into the directory, then ```git submodule update --init --recursive``` will fetch the dependencies, (at least this method works for me), no need to manually copy the dependencies into the source folder.

Reply:
In the new Git Desktop (Atom-based) it automatically installs dependent repos as part of the initial cloning, so long as those repos exist in the user account of the main repo.

Reply:
I use command line git and msbuild tools (no GUI at all to build the plugin, just a single batch file), so perhaps there's an equivalent recursive clone command